### PR TITLE
Adding new subtest to create n number of namespaces

### PR DESCRIPTION
**Pull request by Naresh-ibm**
current test cases doesn't has any test that can create with given count so adding same to validate.

### CODE REVIEW

File: io/disk/ssd/nvmetest.py [hunk 1]
```diff
@@ -44,6 +45,7 @@ def setUp(self):
         """
         self.device = self.params.get('device', default='nvme0')
         self.device = "/dev/%s" % self.device
+        self.ns_count = self.params.get('namespace_count', default=1)
```
Comment (PraveenPenguin) [inline io/disk/ssd/nvmetest.py:48]: can  this be moved in test itself and having variable scop local

Comment (Naresh-ibm) [reply io/disk/ssd/nvmetest.py:48]: @PraveenPenguin we will be needing this in other test too in future, as complete test structure will be going to change going forward.
So I feel having in setup will help better.

File: io/disk/ssd/nvmetest.py [hunk 2]
```diff
@@ -431,3 +433,10 @@ def testsubsystemreset(self):
         cmd = '%s subsystem-reset %s' % (self.binary, self.device)
         if process.system(cmd, ignore_status=True, shell=True):
             self.fail("Subsystem reset failed")
+
+    def test_create_namespaces(self):
+        """
+        creates the specified number of namespaces on nvme drive
+        """
+        device = self.device.split("/")[-1]
+        nvme.create_n_namespaces(device, self.ns_count)
```
[the remark this answers is not in the document]
Comment (Naresh-ibm) [reply io/disk/ssd/nvmetest.py:442]: @abdhaleegit 
1. delete part is taken care in utils
2. util will error out if creation of any namespace fails

so no extra validation is required here

File: io/disk/ssd/nvmetest.py (hunk 1 above)
Comment (PraveenPenguin) [reply io/disk/ssd/nvmetest.py:48]: no, still I feel whenever requirement we should ACT not on a futuristic patch  :) , current one will be based on today's requirement

File: io/disk/ssd/nvmetest.py (hunk 2 above)
Comment (abdhaleegit) [inline io/disk/ssd/nvmetest.py:442]: @Naresh-ibm please  rename the create_n_namespace to proper one which is in utils

**Comment by abdhaleegit** [comment]
@Naresh-ibm looks like merge conflicts

**Review by abdhaleegit — CHANGES_REQUESTED** [review]
@Naresh-ibm we need one more test to delete the namesapces from cfg

please add test_delete_namspaces()

**Pull request closed without merging**